src/robot/libraries/BuiltIn.py

```diff
@@ -2944,26 +2944,38 @@ def _yield_logged_messages(self, messages):
             else:
                 yield value
 
-    def log_to_console(self, message, stream='STDOUT', no_newline=False):
+    def log_to_console(self, message, stream='STDOUT', no_newline=False, format=''):
         """Logs the given message to the console.
 
         By default uses the standard output stream. Using the standard error
-        stream is possibly by giving the ``stream`` argument value ``STDERR``
+        stream is possible by giving the ``stream`` argument value ``STDERR``
         (case-insensitive).
 
         By default appends a newline to the logged message. This can be
         disabled by giving the ``no_newline`` argument a true value (see
         `Boolean arguments`).
 
+        By default adds no alignment formatting. The format argument can be 
+        used to set the alignment of the log message, along with adding 
+        custom padding. Please see 
+        https://docs.python.org/3/library/string.html#formatspec for
+        detailed alignment possibilities. This argument is new in Robot
+        Framework 5.0.
```

Comment:
- A link, not url, would be a bit nicer.
- Argument name should be in double backtics.

Something like this perhaps:

    The ``format`` argument can be used to format the logged message according to the
    [https://docs.python.org/3/library/string.html#formatspec|format specification]. It allows,
    for example, alignment and padding. This argument is new in Robot Framework 5.0.

You can see how the generated documentation looks like by installing `invoke` and then running

    invoke library-docs builtin